### DISCUSSION TITLE
Proper PPM?

### DISCUSSION THREAD

**Discussion by reneselander** [jvde-github/AIS-catcher#185] [Q&A] [answered]
Hello. Using Windows 10 and AIS-catcher with Nooelec NESDR SMArTee ( running 24/7 ) but what about the PPM data? Have been reading a lot about this topic and changed PPM settings in the start.bat ( -p 1 etc... ) but still not able to get the average 0 PPM per hour. I am receiving a good amount of data ( about 50 vessels within a radius of 12 NM ). What is your opinion about the PPM in the screenshot? Is it good enough to keep running without any adjustments? Also, I read that a proper SAW filter ( https://store.uputronics.com/index.php?route=product/product&path=59&product_id=93 ) would be beneficial to this setup, anybody with experience from this kind of filter?
![ppm_screenshot](https://github.com/jvde-github/AIS-catcher/assets/50113423/6662058f-7465-4ce1-9a4c-501f0620454c)

**Comment by jvde-github** [comment, marked as answer]
The PPM looks perfectly fine and no action is needed. Note that this is the shift as calculated by the program to undo any frequency offset in the received signal. This calculation contains noise but also any frequency shift of the sender. So will be difficult to get precisely zero. For modern dongles with frequency stabilization frequency shift is not an issue. Perhaps to add, if the dongle offset  is within -5 to +5 ppm the algorithm will compensate for it without loss of reception. 

On filters, very much depends on location and system. For example, the one you refer to did not make a difference for me. On the other hand the ShipXplorer dongle and the GPIO one did work very well for me. In fact are critical. But others have reported these do not make a difference and even hampered reception. l suspect it depends on the noise in your environment and I don’t have a good rule of thumb. Perhaps others have?

**Reply by reneselander** [reply]
Thanks for your reply. Yes, location and environment are also crucial for the result. I am happy with the info you provided so I will mark this as answered. And yes, it can be interesting also to read what kind of experience other users have about this topic.
